tests/flow/array-filter/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -41,13 +41,15 @@ console.log(filteredItems);
 /* @flow */
 
 function filterItems(items: Array<string | number>): Array<string | number> {
-  return items.map(item => {
-    if (typeof item === \"string\") {
-      return item.length > 2 ? item : null;
-    } else {
-      return item * 10;
-    }
-  }).filter(Boolean);
+  return items
+    .map(item => {
+      if (typeof item === \"string\") {
+        return item.length > 2 ? item : null;
+      } else {
+        return item * 10;
+      }
+    })
+    .filter(Boolean);
```

Comment:
Hm, I actually like how this looked before and tweaked my old code to make it look the way it was before. But I don't know, it's not a big deal. I felt like it was common to do something like

```
items.map(mapFunction).forEach(item => {
   // ...
})
```

And I'm pretty sure your new code will break that up right?